src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -149,6 +168,20 @@ def skip_schema(self, schema: str):
         except BadRequest as err:
             logger.error(err)
 
+    def unskip_schema(self, schema: str):
+        # Removes skip mark from the schema property
+        try:
+            self._sql_backend.execute(
+                f"ALTER SCHEMA {escape_sql_identifier(schema)} UNSET DBPROPERTIES IF EXISTS('{self.UCX_SKIP_PROPERTY}');"
```

Comment:
Maybe add the `hive_metastore`
```suggestion
                f"ALTER SCHEMA hive_metastore.{escape_sql_identifier(schema)} UNSET DBPROPERTIES IF EXISTS('{self.UCX_SKIP_PROPERTY}');"
```

Reply:
That `IF EXISTS` is nice! Learned something new here